### PR TITLE
BGDIINF_SB-1667: added unittest to a case where geoadmin_variant with…

### PR DESCRIPTION
**Pull request by rebert**
… None will be updated

### CODE REVIEW

File: app/tests/test_collections_summaries.py [hunk 1]
```diff
@@ -173,3 +173,12 @@ def test_update_collection_summaries_asset_update(self):
             "Collection's summaries[proj:epsg] has not been correctly "
             "updated after asset has been inserted."
         )
+
+    def test_update_collection_summaries_none_variant(self):
+        # update a variant, that as been None as a start value
+        collection = self.data_factory.create_collection_sample().model
+        item = self.data_factory.create_item_sample(collection=collection, name='base-bbox').model
+        asset = self.add_asset(item, 'har', None, None, None)
```
Comment (ltshb) [inline app/tests/test_collections_summaries.py:181]: An asserttion before that check the collection summaries would be good. Also then an assertion after the add asset that the collection summaries are still empty would be good.

File: app/tests/test_collections_summaries.py [hunk 2]
```diff
@@ -173,3 +173,12 @@ def test_update_collection_summaries_asset_update(self):
...
+        asset.geoadmin_variant = "krel"
+        asset.full_clean()
+        asset.save()
```
Comment (ltshb) [inline app/tests/test_collections_summaries.py:184]: Finally an assertion at the end that check the collection summaries with the "krel" variant would be good

**Review by ltshb — CHANGES_REQUESTED** [review]

File: app/tests/test_collections_summaries.py (hunk 2 above)
Comment (rebert) [reply app/tests/test_collections_summaries.py:184]: ha - I forgot assertions :-) But it works without 500er ...

**Review by ltshb — APPROVED** [review]

**Review by IsabelleBzr — APPROVED** [review]

**Review by hansmannj — APPROVED** [review]
:+1:

**Pull request merged**